dist/common/scripts/scylla_setup

```diff
@@ -169,11 +169,36 @@ if __name__ == '__main__':
     if not is_nonroot() and os.getuid() > 0:
         print('Requires root permission.')
         sys.exit(1)
+
+    if is_offline():
+        default_no_ntp_setup = True
+        default_no_node_exporter = True
+        default_no_kernel_check = False
+        default_no_raid_setup = False
+        default_no_coredump_setup = False
+        default_no_cpuscaling_setup = False
+
+        if not shutil.which('mkfs.xfs'):
+            default_no_kernel_check = True
+            default_no_raid_setup = True
+
+        if not shutil.which('mdadm'):
+            default_no_raid_setup = True
+
+        if not shutil.which('coredumpctl'):
+            default_no_coredump_setup = True
+
+        if is_debian_variant() and not shutil.which('cpufreq-set'):
+            default_no_cpuscaling_setup = True
+        elif not shutil.which('cpupower'):
+            default_no_cpuscaling_setup = True
+
```

Comment:
We should print something for what we skip, so the user is aware the installation is partial.

Let's also create a setup log (not in this series) so the user can report problems easily.

Reply:
Added messages to warn some setup are skipped.